src/redux/reducers/dictionaryReducers.js

```diff
@@ -1,6 +1,8 @@
-import { FETCHING_DICTIONARIES, IS_FETCHING, FETCHING_DICTIONARY, CLEAR_DICTIONARY } from '../actions/types';
+import { FETCHING_DICTIONARIES, IS_FETCHING, FETCHING_DICTIONARY, CLEAR_DICTIONARY, FETCH_VERSIONS } from '../actions/types';
```

Comment:
@kmunene  It would be nicer to maintain the present continuous tense used. `FETCHING_VERSIONS`. What do you think?

Reply:
I totally agree. Will change it. Thanks.